docs/current_docs/manuals/user/cloud/572923-get-started.mdx

```diff
@@ -235,9 +231,33 @@ git commit -a -m "Updated README"
 git push
 ```
 
-### Visualize a run
+### Visualizing Dagger pipelines
+
+:::info
+Dagger 0.11.0 adds support for visualizing Dagger Functions calls in Dagger Cloud. With this update, Dagger Cloud will default to the use the new Traces experience for both `dagger call` and `dagger run` commands. Engines running Dagger 0.11.0 or later will automatically use the new Traces experience. Engines running Dagger 0.10.0 or earlier will continue to use the legacy Runs experience.
```

Comment:
Question, is it `Traces` or `traces`? I see both styles used, should we fix on one?

Reply:
Capital T